allennlp/modules/seq2seq_encoders/pytorch_seq2seq_wrapper.py

```diff
@@ -29,76 +29,93 @@ class PytorchSeq2SeqWrapper(Seq2SeqEncoder):
     Note that we *require* you to pass sequence lengths when you call this module, to avoid subtle
```

Comment:
Just noticed that this is out-dated - you don't pass sequence lengths, you pass a mask.